gusto/numerical_integrator.py

```diff
@@ -0,0 +1,56 @@
+import numpy as np
+
+
+class NumericalIntegral(object):
+    """
+    A class for numerically evaluating and tabulating some 1D integral.
+    :arg lower_bound: lower bound of integral
```

Comment:
These docstrings don't match the style now used through the rest of Gusto -- would you be able to be able to update this?